docker-compose.yml

```
@@ -0,0 +1,9 @@
+version: "3"
+services:
+  app:
+    build: 
+      dockerfile: Dockerfile
+      context: .
```

I suspect this worked _before_ you changed to a multistage Dockerfile, but now it builds the dev image (since that's the last one, it becomes the default) so we should add an explicit target
```suggestion
      context: .
      target: production
```